tchannel/tornado/tchannel.py

```diff
@@ -212,6 +213,12 @@ def request(self,
 
             Defaults to 'c'.
         """
+        # TODO: do not warn if being called from tchannel.TChannel
```

Comment:
@abhinav :P - I don't feel strongly either way, was just sketchin' out what I think we want. At this stage, I'm looking for ideas on how to accomplish deprecating these properly instead of just "dont do this".